lib/main.coffee

```diff
@@ -49,8 +38,8 @@ module.exports =
         filePath = textEditor.getPath()
         command = @executablePath
         ruleset = @rulesets
-        if @projectRules
-          rulesetPath = helpers.find(filePath, 'ruleset.xml')
+        if /^[a-z0-9]+\.xml$/gi.test(@rulesets)
```

Comment:
Since it's a major release I'm not too worried about migration, but it wouldn't be that hard. Something along the lines of running

```js
if (atom.config.get('linter-phpmd.projectRules') !== undefined):
  atom.notifications.addInfo('The automatic searching for ruleset.xml has been removed. To regain that functionality please add ruleset.xml to the Rulesets setting.')
```

The conditional probably isn't perfect, and the wording could use a bit of work, but that's close :stuck_out_tongue:.